open_instruct/dpo_tune.py

```diff
@@ -1034,6 +1038,27 @@ def load_model():
                 location=args.hf_repo_id,
                 hf_repo_revision=args.hf_repo_revision,
             )
+    if args.hf_metadata_dataset:
+        # dpo script only supports these two options right now for datasets
+        dataset_name = args.dataset_name if args.dataset_name else args.train_file
+        beaker_config = maybe_get_beaker_config()
+        # mainly just focussing here on what would be useful for the leaderboard.
+        # wandb will have even more useful information.
+        metadata_blob = {
+            "model_name": args.exp_name,
+            "model_type": "sft",
+            "datasets": [dataset_name],
+            "base_model": args.model_name_or_path,
+            "wandb_path": wandb_tracker.run.get_url(),
+            "beaker_experiment": beaker_config.beaker_experiment_url,
+            "beaker_datasets": beaker_config.beaker_dataset_id_urls
+        }
+        upload_metadata_to_hf(
+            metadata_blob,
+            "metadata.json",
+            args.hf_metadata_dataset,
+            'results/' + args.hf_repo_revision,  # to match what the auto-evals name as.
+        )
```

Comment:
```suggestion
    if args.hf_metadata_dataset and accelerator.is_main_process and is_beaker_job():
        # dpo script only supports these two options right now for datasets
        dataset_name = args.dataset_name if args.dataset_name else args.train_file
        # mainly just focussing here on what would be useful for the leaderboard.
        # wandb will have even more useful information.
        metadata_blob = {
            "model_name": args.exp_name,
            "model_type": "sft",
            "datasets": [dataset_name],
            "base_model": args.model_name_or_path,
            "wandb_path": wandb_tracker.run.get_url(),
            "beaker_experiment": beaker_config.beaker_experiment_url,
            "beaker_datasets": beaker_config.beaker_dataset_id_urls
        }
        upload_metadata_to_hf(
            metadata_blob,
            "metadata.json",
            args.hf_metadata_dataset,
            'results/' + args.hf_repo_revision,  # to match what the auto-evals name as.
        )
```

`beaker_config` is already available above
https://github.com/allenai/open-instruct/blob/e77830ed5409eb4862ce5b4ad61ba560ae8d8a10/open_instruct/dpo_tune.py#L843